int32 iamhere
geometry_msgs/PoseStamped robotPose
---
int32 ack
int32 gonext
